packages/service-worker-mock/models/NavigationPreloadManager.js

```diff
@@ -0,0 +1,29 @@
+const PushSubscription = require('./PushSubscription');
+
+// https://developer.mozilla.org/en-US/docs/Web/API/NavigationPreloadManager
+class PushManager {
```

Comment:
Oops, forgot I had this locally. It's a stub so should be fine since it's already published...